src/Symfony/Component/Serializer/Exception/ExtraAttributesException.php

```diff
@@ -18,10 +18,35 @@
  */
 class ExtraAttributesException extends RuntimeException
 {
+    /**
+     * Extra attributes.
+     *
+     * @var array
```

Comment:
Same. The constructor already typehints the `$extraAttributes` argument, and the name is self explanatory :)